tests/acceptance/suites/themeupgrader.js

```
@@ -0,0 +1,27 @@
+const process = require('process');
+const git = require('simple-git/promise')();
+const { runInPlayground } = require('../setup/playground');
+const ThemeManager = require('../../../src/utils/thememanager');
+const path = require('path');
+
+ThemeManager.getRepoForTheme = () => {
+    return path.resolve(__dirname, '../test-themes/basic-flow');
+};
+
+const error = console.error;
+console.error = jest.fn();
+afterAll(() => console.error = error);
```

Can you add a comment to explain why this was needed? Is it because the upgrade branch fail would otherwise output a bunch of error messages when running tests?

---

upgrade --branch fail outputs an error message when we run tests. The tests still pass, we just thought it would be cleaner if they didn't display error messages when we run them.
I'll specify that it's to silence errors.